akasha-vladan-main/css/style.scss

```diff
@@ -0,0 +1,76 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap');
+
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'Open Sans', sans-serif;
+
+}
```

Comment:
I think it would be better to properly indent your rules so as to make the code look better and to avoid css linter errors